ldap-client.go

```diff
@@ -202,3 +202,28 @@ func (lc *LDAPClient) GetGroupsOfUser(username string) ([]string, error) {
 	}
 	return groups, nil
 }
+
+// GetLdapGroups returns list of groups matching a name.
+func (lc *LDAPClient) GetLdapGroups(groupName string) ([]string, error) {
```

Comment:
I would change to getAllGroups as it is obvious that it's ldap.

Reply:
Changed to GetAllGroupsByName